.circleci/config.yml

```diff
@@ -599,11 +599,11 @@ workflows:
             tags:
               only: /^v[0-9]+\.[0-9]+\.[0-9]+/
             branches:
-              ignore: /.*/
+              only: dev
```

Comment:
1. It seems that `branches` key is deprecated https://circleci.com/docs/2.0/configuration-reference/#branches--deprecated
2. Don't we need to publish docker with `master` branch?

Reply:
on `jobs:` yes, but it seems okay since we use this under workflows --> https://circleci.com/docs/2.0/workflows/

it trigger by tag like `v1.6.0` so that we don't need to use branch when it comes to major version release.